webdriver/tests/bidi/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Compares 2 objects recursively ignoring values of specific attributes.
+def recursive_compare(expected, actual, ignore_attributes=None):
+    if ignore_attributes is None:
+        ignore_attributes = []
+    assert type(expected) == type(actual)
+    if type(expected) is list:
+        assert len(expected) == len(actual)
+        for index, val in enumerate(expected):
+            recursive_compare(expected[index], actual[index], ignore_attributes)
+        return
+
+    if type(expected) is dict:
+        assert expected.keys() == actual.keys(), \
```

Comment:
`actual` can usually have more keys as part of the forwards-compat design.

Reply:
Valid point, done.